scripts/prepare_datasets/prepare_datasets.py

```diff
@@ -77,8 +74,30 @@
     lines.append(f'| {sortid} | [view]({le_url}) | {description} |')
 lines.append('')
 
+lines.append('')
+lines.append(f'[Browse all recordings]({aws_url}/default?feed={x.get_uri()})')
+
 txt = '\n'.join(lines)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 print(txt)
 print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 
+try:
+    with open('README.md') as f:
+        readme_txt = f.read()
+    ind1 = readme_txt.find('<!-- BEGIN DATA TABLE -->')
+    ind2 = readme_txt.find('<!-- END DATA TABLE -->')
+    assert (ind1 >=0) and (ind2 >= 0)
```

Comment:
might be worthwhile to assert that ind2 >= ind1?

Reply:
good idea